.github/workflows/CI.yaml

```diff
@@ -29,3 +29,8 @@ jobs:
           check_name: Test Results
           fail_on_test_failures: true
           fail_if_no_tests: false
+      - name: Validate openapi spec
+        if: ${{ always() }}
+        run: |
+          differences=$(diff openapi/openapi.yaml manager/target/generated/openapi.yaml)
+          if [ $? -ne 0 ]; then echo "Please move the manager/target/generated/openapi.yaml into openapi/openapi.yaml"; false ; fi
```

Comment:
Just to double check, is this step enough to make the build fail?

Reply:
Yep, `diff` will return status code `1` if the two files are different. Then the step will fail and the build would be red